corehq/apps/toggle_ui/views.py

```diff
@@ -200,7 +200,10 @@ def _get_usage_info(toggle):
                 active_domains += 1
         else:
             try:
-                last_used[name] = _format_date(CouchUser.get_by_username(name).last_login)
+                user = CouchUser.get_by_username(name)
+                if not user:
+                    raise ResourceNotFound
+                last_used[name] = _format_date(user.last_login)
```

Comment:
Better.

Also on a different note, `CouchUser.get_by_username` could take a raise_if_none kwarg. I was tempted to do that and refactor wherever it is being used. But it didn't seem like a good enough change. What do you think